.changeset/big-owls-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": patch
+---
+
+Fix table empty state horizontal spacing when table is very narrow.
```

Comment:
To confirm, is it "spacing" or "alignment" that has changed?

Reply:
Updated to use "alignment".